### PR TITLE
Components

### PR DESCRIPTION
**Pull request by levy002**
## Description
In this PR, we created the Calculator components.

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### Milestone requirements, We:

- [x] deleted all the boilerplate from CRA (text, images, styles).
- [x] created a directory called ``components``. 
- [x] create a new ``calculator.js`` file inside ``components`` . 
- [x] developed the UI of the calculator.
- [x] used `class` based components. 
- [x] imported it and used it in our main component `App.js`.

 # Happy reviewing 😃

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,6 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```
Comment (mikethreels) [inline src/App.js:4]: - [ ] according to the requirements you should make use of class-based components, please make sure The App.js component is also class-based.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Levy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (levy002) [reply src/App.js]: Thank you! @mikethreels.  I've implemented the changes as requested.

**Review by mikethreels — APPROVED** [review]
Hi Levy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**